ansible_collections/arista/avd/roles/eos_designs/templates/network_services/ethernet-interfaces.j2

```diff
@@ -33,6 +33,25 @@ ethernet_interfaces:
 {%                         if l3_interface.description is arista.avd.defined %}
     description: {{ l3_interface.description }}
 {%                         endif %}
+{%                         if l3_interface.ospf.enabled is arista.avd.defined(true) %}
```

Comment:
add a check if `vrf.ospf.enabled is arista.avd.defined(true)`

Reply:
Fixed.